tiny_dnn/tiny_dnn.h

```diff
@@ -84,6 +86,13 @@ using ave_pool = tiny_dnn::average_pooling_layer;
 
 using fc = tiny_dnn::fully_connected_layer;
 
+// TODO(prlz77) RNN Wrapper
+// template<class T>
+// using recurrent = tiny_dnn::recurrent_layer<T>;
+
+template <class activation_type>
+using recurrent_cell = tiny_dnn::recurrent_cell_layer<activation_type>;
```

Comment:
Oh I didn't know about that. Thank you for telling me that.

But still I think you should probablly pass an activation function layer instance via variable argument.
If you pass activation function type as a template parameter, serialization of recurrent_cell_layer will not be simple nor easy.